pwa/app/components/tba/teamEventAppearance.tsx

```diff
@@ -0,0 +1,63 @@
+import { Link } from '@remix-run/react';
+
+import BiCalendar from '~icons/bi/calendar';
+import BiPinMapFill from '~icons/bi/pin-map-fill';
+
+import { Event, Match, TeamEventStatus } from '~/api/v3';
+import InlineIcon from '~/components/tba/inlineIcon';
+import MatchResultsTable from '~/components/tba/matchResultsTable';
+import { Badge } from '~/components/ui/badge';
+import { getEventDateString } from '~/lib/eventUtils';
+
+export default function TeamEventAppearance({
+  event,
+  matches,
+  status,
+}: {
+  event: Event;
+  matches: Match[];
+  status: TeamEventStatus | null;
+}): JSX.Element {
+  return (
+    <div className="flex flex-wrap gap-x-8 [&>*]:sm:flex-1">
+      <div className="">
+        <h2 className="text-2xl">
+          <Link to={`/event/${event.key}`}>{event.name}</Link>
+        </h2>
+        <InlineIcon>
```

Comment:
We might want to think about extracting these out into reusable components in the future. not in this PR